scripts/image_tools/images/hdr.py

```diff
@@ -19,9 +19,11 @@
 
```

Comment:
We should probably add the python3 shebang

Reply:
done

Reply:
Was it really done? It doesn't seem so...

Reply:
The Python 3 shebang has been added to the `scripts/image_tools/*py` scripts which are designed to be executed. Adding a shebang to `hdr.py` which is a Python file used by the scripts is meaningless.

So this seems to be correctly solved, yes.